src/resources/texts.tsx

```diff
@@ -2,6 +2,79 @@ import React from 'react'
 
 const PLACEHOLDER = 'XXX'
 
+export const TEXT_KEYS = {
```

Comment:
I didn't read about it anywhere but I just felt it would be better than typing long strings with little to no auto-complete support for keys. Putting them all in TEXT_KEYS seemed like a quicker approach for future additions too.

Reply:
since we use typescript, there should be autocomplete for strings when using `i11n`